icaruscode/PMT/Trigger/TriggerSimulationOnGates_module.cc

```diff
@@ -889,8 +923,20 @@ icarus::trigger::TriggerSimulationOnGates::TriggerSimulationOnGates
     log << "\nConfigured " << fADCthresholds.size() << " thresholds (ADC):";
     for (auto const& [ thresholdTag, dataTag ]: fADCthresholds)
       log << "\n * " << thresholdTag << " (from '" << dataTag.encode() << "')";
+#if 0 // TODO restore after adoption of https://github.com/LArSoft/lardataalg/pull/44
     log << "\nOther parameters:"
-      << "\n * trigger time resolution: " << fTriggerTimeResolution;
```

Comment:
Did that PR get adopted? Should we restore?

Reply:
Oops, this was intended for line 926: `#if 0 // TODO restore after adoption of https://github.com/LArSoft/lardataalg/pull/44`

Reply:
I think so. However, if you don't mind, I would postpone that to the next PR (which will come soon anyway, since I have changes to make trigger efficiency trees for MC to be merged, and probably more adder stuff too).

Reply:
Unless I have to update this PR to the latest LArSoft/icaruscode release, in which case I will also fix this at the same time.